src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package kitchenpos.application;
+
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.fake.menuGroup.TestMenuGroupRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class MenuGroupServiceTest {
+    @Test
+    @DisplayName("메뉴그룹을 생성할 수 있다")
+    void create_menu_group() {
+        MenuGroupService menuGroupService = new MenuGroupService(new TestMenuGroupRepository());
+        String menuGroupName = "TestMenuGroup";
+        MenuGroup request = new MenuGroup();
+        request.setName(menuGroupName);
+        MenuGroup menuGroup = menuGroupService.create(request);
+
+        assertAll(
+                () -> assertThat(menuGroup.getId()).isNotNull(),
+                () -> assertThat(menuGroup.getName()).isEqualTo(menuGroupName)
+        );
+    }
+
+    @ParameterizedTest
+    @DisplayName("메뉴그룹이름은 존재해야하며 길이가 1 이상이다")
+    @NullSource
+    void create_menu_group_fail(String menuGroupName) {
+        MenuGroupService menuGroupService = new MenuGroupService(new TestMenuGroupRepository());
+        MenuGroup request = new MenuGroup();
+        request.setName(menuGroupName);
+
+        assertThrows(IllegalArgumentException.class, () -> menuGroupService.create(request));
+    }
+
+    @Test
+    @DisplayName("메뉴그룹을 조회할 수 있다")
+    void find_menu_group() {
+        // given
+        MenuGroupService menuGroupService = new MenuGroupService(new TestMenuGroupRepository());
+        String menuGroupName = "TestMenuGroup";
+        MenuGroup request = new MenuGroup();
```

Comment:
이왕이면 미리 만들어 놓으신 fixture를 이용하면 좋을 것 같아요.
```suggestion
        MenuGroup request = createMenuGroup();
```

Reply:
Fixture의 변수명을 아래의 예시와 같이 한글로 의미있게 정의하면
처음엔 어색할 수 있으나 직관성을 높여 테스트 케이스에 대한 가독성을 높일 수도 있습니다. 😄
테스트 데이터도 가능한 실제 데이터와 흡사하게 넣어주는 것이 좋구요.
```
val 한마리_메뉴그룹 = createMenuGroup("한마리 메뉴");
```

Reply:
변경했습니다